Testing/SystemTests/tests/framework/PowderReduceP2DTest.py

```diff
@@ -55,6 +56,10 @@ def validateMethod(self):
     def validate(self):
         return self.outputFile + '.p2d', self.reference
 
+    def skipTests(self):
+        # If CONDA_PREFIX is defined skip tests
```

Comment:
Could we put a comment why this is skipped on Conda?
Also I'm not comfortable with these broad skip clauses, since they get forgotten whilst the original developer(s) assume they are still running once we migrate fully to Conda

Reply:
Tracked here: https://github.com/mantidproject/mantid/issues/32088